pdudaemon/drivers/ykush.py

```diff
@@ -52,19 +54,37 @@ def port_interaction(self, command, port_number):
             raise RuntimeError(err)
 
         if command == "on":
-            byte = 0x11
+            byte = 0x10 + port_number
         elif command == "off":
-            byte = 0x01
+            byte = 0x00 + port_number
         else:
             log.error("Unknown command %s." % (command))
             return
 
         d = hid.device()
-        d.open(YKUSH_VID, YKUSH_XS_PID, serial_number=self.serial)
+        d.open(YKUSH_VID, self.ykush_pid, serial_number=self.serial)
         d.write([byte, byte])
         d.read(64)
         d.close()
 
     @classmethod
     def accepts(cls, drivername):
-        return drivername in cls.supported
+        return False
```

Comment:
If you keep this one functional rather then dummy (doesn't really have to be a dummy here in the first place) then you would just have to set support = [..]. in the classes that inherent rather then reimplementing the accepts method.

I guess you could even make accepts just be `drivername == cls.__name__.upper()`